python/cucim/src/cucim/core/operations/color/tests/test_color_jitter.py

```diff
@@ -0,0 +1,18 @@
+import cucim.core.operations.color as ccl
+import numpy as np
+import cupy
+from PIL import Image
+
+img = Image.open('1.png')
```

Comment:
You may want to something like this:
```suggestion
img = Image.open(os.path.join(os.path.abspath(os.path.dirname(__file__)), "1.png"))
```

Reply:
Included as suggested.